# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/RemoteVideoDecoder.h
# Commit: e1521d8fbe90
# Full Hash: e1521d8fbe9007e783ea1c094b0d3e5574d0fbf1
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-06-27 09:34:48
# Regressor Bug: 1561178
# File Overlap Count: 1
# Description:
#   Bug 1561178 - Combine the two video RecvOutput methods and switch based on the descriptor type. r=jya
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D35964
# ==============================================================================

diff -r 5ffc2eb8ac9f -r e1521d8fbe90 dom/media/ipc/RemoteVideoDecoder.h
--- a/dom/media/ipc/RemoteVideoDecoder.h	Thu Jun 27 05:59:05 2019 +0300
+++ b/dom/media/ipc/RemoteVideoDecoder.h	Thu Jun 27 01:28:47 2019 +0000
@@ -45,8 +45,6 @@
                        const CreateDecoderParams::OptionSet& aOptions,
                        const layers::TextureFactoryIdentifier& aIdentifier);
 
-  IPCResult RecvOutput(const DecodedOutputIPDL& aDecodedData) override;
-
   void RecordShutdownTelemetry(bool aAbnormalShutdown) override;
 
  private:
